proxy/InkAPI.cc

```diff
@@ -9212,6 +9213,19 @@ TSUuidStringGet(const TSUuid uuid)
   return nullptr;
 }
 
+const char *
+TSClientRequestUuidGet(TSHttpTxn txnp)
+{
+  TSUuid process = TSProcessUuidGet();
+  if (process) {
+    std::ostringstream oss;
+    oss << TSUuidStringGet(process) << '-' << TSHttpTxnIdGet(txnp);
+    return oss.str().c_str();
```

Comment:
@bryancall agreed, maybe I should not use ostringstream at all to avoid that all together.

@jpeach care to elaborate? I'm not completely sure what you mean with that.